Assets/Scripts/InteractableItem.cs

```diff
@@ -14,4 +14,10 @@ public virtual bool Enabled()
     {
         return true;
     }
+    public virtual List<Sprite> GetSprite(PlayerInteractionController Caller)

```

Comment:
I suggest returning `IEnumerable<Sprite>`, just in case.  Item display itself accepts arbitrary collections.

Reply:
Yeah, good point